leakcanary-object-watcher-android/src/main/java/leakcanary/AppWatcher.kt

```diff
@@ -185,6 +185,9 @@ object AppWatcher {
   val objectWatcher
     get() = InternalAppWatcher.objectWatcher
 
+  val isDebuggableBuild
```

Comment:
This is not pretty. But it was the easiest way I could think of to make `InternalAppWatcher.isDebuggableBuild` accessible to `InternalLeakCanary`.